### PR TITLE
AB2-68 Add parallel building of APKs to CI

### PR DESCRIPTION
**Pull request by joseph-cliqz**

### CODE REVIEW

File: mozconfigs/ghostery.mozconfig [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Build Ghostery for Android:
+
+mk_add_options GIT=/usr/bin/git
+
+# Branding
+ac_add_options --with-branding=mobile/android/branding/ghostery
+
+
+# Build Firefox for Android Artifact Mode:
+ac_add_options --with-android-min-sdk=21
```
Comment (ravjit-cliqz) [inline mozconfigs/ghostery.mozconfig]: forgot to remove?

File: Jenkinsfile [hunk 2]
```diff
@@ -1,131 +1,174 @@
...
+                        sh """#!/bin/bash -l
                             set -e
                             set -x
-                            cp mozconfigs/cliqz.mozconfig mozilla-release/mozconfig
+                            rm -rf autobot*
+                            cp ${flavor} mozilla-release/mozconfig
                             cd mozilla-release
+                            ./mach clobber
                             ./mach build
-                            for lang in `ls ../l10n/`; do
```
Comment (spacifici) [inline Jenkinsfile]: Why did you remove this? This is mandatory for multi-language builds

Comment (joseph-cliqz) [reply Jenkinsfile]: Will put it back

File: mozconfigs/ghostery.mozconfig (hunk 1 above)
Comment (joseph-cliqz) [reply mozconfigs/ghostery.mozconfig]: Will remove
